components/ActionSheet/examples/ActionSheetTypicalUseExampleViewController.m

```diff
@@ -25,51 +29,37 @@ @interface ActionSheetTypicalUseExampleViewController ()
 
 @end
 
-@implementation ActionSheetTypicalUseExampleViewController {
-  MDCButtonScheme *_buttonScheme;
-}
+@implementation ActionSheetTypicalUseExampleViewController
 
 - (instancetype)init {
   self = [super init];
   if (self) {
     self.title = @"Action Sheet";
-    _colorScheme =
-        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-    _typographyScheme = [[MDCTypographyScheme alloc] init];
+    _containerScheme = [[MDCContainerScheme alloc] init];
```

Comment:
Please change the property type above to be `id<MDCContainerScheming>` to prevent accidentally overwriting properties.

Reply:
Fixed